python-mappings/README.md

```diff
@@ -0,0 +1 @@
+Code defining the `PizzaMenu` class in the tutorial Python Mappings: A Comprehensive Guide
```

Comment:
Let's use the same format of the README file as in other tutorials for consistency.